docs/libraries/how-to-run-the-website-in-your-local-server.md

```diff
@@ -44,21 +39,21 @@ cd <arrow-library>
 ./gradlew buildArrowDoc
 ```
 
-Then copy the result to the previous directory:
+Then copy the result to the `docs` directory:
 
 ```sh
-cp -r <arrow-library>/arrow-docs/build/site/* arrow-site/build/site/
+cp -r <arrow-library>/arrow-docs/build/site/* arrow-site/docs
```

Comment:
Can this step be automated?

Reply:
Yes, it's already automated: 

```
./gradlew runArrowSite
```

Just showing an alternative way with manual steps.